whelktool/scripts/examples/mergeHoldings.groovy

```diff
@@ -0,0 +1,128 @@
+import whelk.Document
+
+/********* Settings section, you will need to provide values for these variables **********/
+
+// Define your list of instances for which to merge holdings, you probably want one of
+// these forms:
```

Comment:
Would be good to say here that the old id form (e.g. 11904213 instead of r934fz030k4wj0p) won't do. Unless the old id form is handled by the script.